### PR TITLE
Update protoc-installation.md

### PR DESCRIPTION
**Pull request by fasterinnerlooper**
Add Windows installation instructions.

### CODE REVIEW

**Comment by ejona86** [comment]
@veblush, do you know who should review this?

**Comment by veblush** [comment]
Sorry for my late review. LGTM.

**Review by veblush — APPROVED** [review]

File: content/en/docs/protoc-installation.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ Linux or macOS using the following commands.
   $ brew install protobuf
   $ protoc --version  # Ensure compiler version is 3+
   ```
+- Windows, using [Winget][]
```
Comment (veblush) [inline content/en/docs/protoc-installation.md:45]: I think `[Winget][]` should be `[Winget][winget]` to give it a proper link?

**Review by shivaspeaks — COMMENTED** [review]
LGTM

**Review by shivaspeaks — APPROVED** [review]

**Pull request merged**